convert_osm/src/split_ways.rs

```diff
@@ -249,9 +250,8 @@ fn dedupe_angles(pts: Vec<Pt2D>) -> Vec<Pt2D> {
         {
             result.pop();
             result.push(pt);
-        } else {
-            result.push(pt);
         }
+        result.push(pt);
```

Comment:
This looks like a change in logic, right? Since we always push the pt last, maybe we could just look for that `pop` case and do that, and always push.

Reply:
Yes, I messed that one up. Will fix!

Reply:
Should be fine now, but please double check!

(I highly suspect that this was causing my build to crash when trying to add new cities...)

Reply:
New logic is simpler and equivalent. Thanks!

Is there a clippy-specified preference between `let mut result = Vec::<Pt2D>::new()` and `let mut result: Vec<Pt2D> = Vec::new()`? We're generally using the latter, but actually the former is less repetitive and feels more like English. I'll write down a TODO to later switch more of the code-base to this style, for consistency

Reply:
No, that was just my choice. I liked to avoid repeating `Vec`.

I'll change it to the latter.

Reply:
Done.